website/docs/downloads/windows.md

```diff
@@ -25,6 +25,21 @@ the changes and improvements introduced by this release.
     If Windows 8.x or Windows 10 prevents you from running DOSBox Staging via
     Microsoft Defender SmartScreen, see the [section below](#microsoft-defender-smartscreen).
 
+For 32-bit Windows 7 and newer, and for 64-bit Windows 7 hosts use the respective MSVC builds:
+- The 32-bit MSVC build [DOSBox Staging 0.80.1 32-bit (zip)][0_80_1_x32_ZIP]
+- The 64-bit MSVC build on Windows 7 or newer can be optionally selected in the installer.
```

Comment:
This section is confusing. Please structure the sections logically as follows:

- Windows 8, 10 & 11  (current instructions)
- Windows 7
- WIndows Vista

For the Windows Vista section, including build commands in this general Download page intended for general non-developer users is too much. Instead, update the Windows specific build instructions intended for devs, and just link to it form here, e.g.

Subsequent releases can be built from source by excluding FluidSynth and Slirp. Please see the Windows build instructions for further details.

(There was also a typo here: built from source, not build)